test/numbers.jl

```diff
@@ -2771,7 +2771,8 @@ let types = (Base.BitInteger_types..., BigInt, Bool,
     for S in types
         for op in (+, -)
             if S === Float16 # type instability here?
-                @test_broken @inferred Base.promote_op(op, S)
+                # broken, fixme then remove this branch
```

Comment:
The issue is that `@test_broken` is currently a replacement of `@test`. So if `@test foo` is broken, you replace it with `@test_broken foo`. However, here `@inferred` is the `@test` so using `@test_broken` or `@test_throws` like this is effectively writing `@test_throws @test` or `@test_broken @test`. I think we could possibly make this a valid use of `@test_broken` but this is almost certainly not a valid use of `@test_throws`.